packages/react-reconciler/src/ReactFiberDevToolsHook.new.js

```diff
@@ -78,18 +84,19 @@ export function onScheduleRoot(root: FiberRoot, children: ReactNodeList) {
   }
 }
 
-export function onCommitRoot(
-  root: FiberRoot,
-  priorityLevel: ReactPriorityLevel,
-) {
+export function onCommitRoot(root: FiberRoot, priorityLevel: LanePriority) {
   if (injectedHook && typeof injectedHook.onCommitFiberRoot === 'function') {
     try {
       const didError = (root.current.flags & DidCapture) === DidCapture;
       if (enableProfilerTimer) {
+        const schedulerPriority =
+          priorityLevel === NoLanePriority
+            ? NormalPriority
+            : lanePriorityToSchedulerPriority(priorityLevel);
```

Comment:
DevTools is expecting the scheduler priority, so I'm converting to it here. I'm not sure what the plan should be for this - i.e should we update devtools to use the updateLanePriority instead of the scheduler priority? cc @bvaughn for interest